website/source/docs/post-processors/vsphere-template.html.md

```diff
@@ -0,0 +1,58 @@
+---
+description: |
+    The Packer vSphere Template post-processor takes an artifact from the VMware-iso builder
+    and allows to mark a VM as a template and leaving it in a path of choice. 
+layout: docs
+page_title: 'vSphere Template - Post-Processors'
+sidebar_current: 'docs-post-processors-vSphere-template'
+---
+
+# vSphere Template Post-Processor
+
+Type: `vsphere-tpl`
```

Comment:
I think it's better if this would be named `vsphere-template`. @mwhooker @SwampDragons what do you think?